README.md

```diff
@@ -35,7 +35,7 @@ Option      | Default                           | Notes
 ------------|-----------------------------------|-------
 `contracts` | —                                 | The path to the solidity contracts to mock
 `out`       | `./out`                           | The path that has the compiled artifacts
-`genDir`    | `./solidity/test/mock-contracts`  | The path to the generated mock contracts
+`genDir`    | `./solidity/test/mocks`           | The path to the generated mock contracts
```

Comment:
Let's also change the name of the parameter. What do you think about `mocks`? So the command would look like
```sh
yarn smock-foundry --contracts solidity/contracts --mocks solidity/test/mocks --out out
```

Reply:
There's no `smock-foundry` command in the `package.json`. Instead, the analogous one is:
```json
"mock-gen": "ts-node src/run.ts mock-gen"
```
Is this intentional?

Reply:
@0xJabberwock yeah, I mean when you install the thingie on a new project and run `yarn smock-foundry` it will work due to the `bin` setting in `package.json`. Idk what `mock-gen` is, probably a relict of the earlier approaches.